src/entities/note.cpp

```diff
@@ -2138,9 +2155,7 @@ QString Note::textToMarkdownHtml(QString str, const QString& notesPath,
     QString codeBackgroundColor = darkModeColors ? QStringLiteral("#444444") : QStringLiteral("#f1f1f1");
 
     // do some more code formatting
-    // the "pre" styles are for the full-width code block background color
     codeStyleSheet += QString(
-                    "pre { display: block; background-color: %1 } "
```

Comment:
I don't think you want to remove that, do you?

Reply:
Nah, leftover. I will fix that.